cmd/thanos/sidecar.go

```diff
@@ -69,6 +72,7 @@ func runSidecar(
 	tracer opentracing.Tracer,
 	reloader *reloader.Reloader,
 	comp component.Component,
+	requestLoggingDecision string,
```

Comment:
```suggestion
	reqLogDecision string,
```
Can we compact this var for readability everywhere? (:

Reply:
Sure!